tests/test_all_rules.py

```diff
@@ -455,6 +456,30 @@ def test_required_lookback(self):
             err_msg = f'The following rules should have a longer `from` defined, due to indexes used\n {rules_str}'
             self.fail(err_msg)
 
+    def test_eql_lookback(self):
+        """Ensure EQL rules lookback => max_span, when defined."""
+        unknowns = []
+        invalids = []
+
+        for rule in self.all_rules:
+            if rule.contents.data.type == 'eql' and rule.contents.data.max_span:
+                if rule.contents.data.look_back == 'unknown':
+                    unknowns.append(self.rule_str(rule, trailer=None))
+                else:
+                    look_back = rule.contents.data.look_back.as_milliseconds()
+                    max_span = rule.contents.data.max_span.as_milliseconds()
+
+                    if look_back < max_span:
```

Comment:
I think we also want to check that the interval is also adjusted.
We don't want to run a 3 hour wide query every 10 minutes.

I think that's a valid test that can be done separately, regardless of EQL

Reply:
ohhh good catch, `interval`

Reply:
tests added

rule updates in #1362